src/connectionconfig/connectionconfig.ts

```diff
@@ -184,7 +184,7 @@ export class ConnectionConfig implements IConnectionConfig {
         groups = groups.filter((group) => group.name === ConnectionConfig.RootGroupName);
 
         if (groups.length === 0) {
-            this._logger.error(
+            this._logger.verbose(
```

Comment:
This should stay as an error; if it can't find a root group, this should is properly an error.